docs/openurl/outline.md

```diff
@@ -287,6 +287,7 @@ genre : "journal|issue|article|proceeding|conference|preprint|unknown" # journal
         "country" : "<country of journal publication>",
         "license" : [<list of titles of licences>],
         "publisher" : "<publisher>",
+        "institution" : "<institution>",
```

Comment:
There's something a bit odd about those old OpenURL docs, we should probably remove the DOAJ models from the 'useful info' section and point them to the main system documentation rather than repeating the models. They could be outdated in other ways.